ckanext/switzerland/tests/test_format_utils.py

```diff
@@ -1,225 +1,238 @@
+        'media_type': None,
+        'format': None
+    }, 'SERVICE'),
+    ({
+        'download_url': 'http://download.url',
+        'media_type': None,
+        'format': 'dogvideo'
+    }, ''),
+    ({
+        'download_url': 'http://download.url',
+        'media_type': None,
+        'format': None
+    }, ''),
+    ({
+        'download_url': None,
+        'media_type': None,
+        'format': 'catgif'
+    }, 'SERVICE'),
+    ({
+        'download_url': None,
+        'media_type': None,
+        'format': 'xml'
+    }, 'XML'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': 'cat/gif',
+         'format': 'gif'
+     }, 'gif'),
+    ({
+         'download_url': None,
+         'media_type': 'html',
+         'format': 'xml'
+     }, 'HTML'),
+    ({
+         'download_url': None,
+         'media_type': 'text/html',
+         'format': 'xml'
+     }, 'HTML'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': None,
+         'format': 'application/vnd.oas...'
+     }, 'ODS'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': None,
+         'format': 'vnd.openxmlformats-officedocument.spreadsheetml.sheet'
+     }, 'XLS'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': None,
+         'format': 'pc-axis file'
+     }, 'PC-AXIS'),
+    ({
+         'download_url': 'http://download.url/Download.aspx?file=pc-axis-file-001',
+         'media_type': 'pc-axis file',
+         'format': 'CSV'
+     }, 'PC-AXIS'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': None,
+         'format': 'Application/Sparql-...'
+     }, 'SPARQL'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': 'application/sparql-query',
+         'format': None
+     }, 'SPARQL'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': None,
+         'format': 'rq'
+     }, 'SPARQL'),
+    ({
+         'download_url': 'http://download.url/foo.sparqlq',
+         'media_type': None,
+         'format': None
+     }, 'SPARQL'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': 'application/ld+json',
+         'format': None
+     }, 'JSONLD'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': None,
+         'format': 'jsonld'
+     }, 'JSONLD'),
+    ({
+         'download_url': 'http://download.url/foo',
+         'media_type': 'text/n3',
+         'format': None
+     }, 'N3'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': None,
+         'format': 'n3'
+     }, 'N3'),
+    ({
+         'download_url': 'http://download.url/foo.n3',
+         'media_type': None,
+         'format': None
+     }, 'N3'),
+    ({
+         'download_url': 'http://download.url',
+         'media_type': 'application/rdf+xml',
+         'format': None
+     }, 'RDF'),
```

Comment:
This test should be than also adapted:  RDF/XML

Reply:
I am not sure about that. From reading the story, I think what we want, is that `application/rdf+xml` is mapped to `RDF`, isn't it @bellisk ?